.obs/dockerfile/micro-rt-iso/Dockerfile

```diff
@@ -1,4 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
+#!ExclusiveArch: x86_64
```

Comment:
This way can prevent the need of repository settings at package level, I truly dislike them as they tend to be forgotten and makes project recreation hard.